src/ensembl/src/content/app/species-selector/components/species-selector-app-bar/SpeciesSelectorAppBar.tsx

```diff
@@ -31,42 +32,34 @@ export const PlaceholderMessage = () => (
 );
 
 export const SpeciesSelectorAppBar = (props: Props) => {
-  return (
-    <div className={appBarStyles.appBar}>
-      <div className={appBarStyles.appBarTop}>
-        <div className={appBarStyles.top}>Species Selector</div>
-      </div>
-      <div className={styles.main}>
-        {props.selectedSpecies.length > 0 ? (
-          <SelectedSpeciesList {...props} />
-        ) : (
-          <PlaceholderMessage />
-        )}
-      </div>
-    </div>
-  );
+  const mainContent =
+    props.selectedSpecies.length > 0 ? (
+      <SelectedSpeciesList {...props} />
+    ) : (
+      <PlaceholderMessage />
+    );
+
+  return <AppBar appName="Species Selector" mainContent={mainContent} />;
 };
 
 const SelectedSpeciesList = (props: Props) => {
   const shouldLinkToGenomeBrowser =
     props.selectedSpecies.filter(({ isEnabled }) => isEnabled).length > 0;
-  return (
-    <>
-      {props.selectedSpecies.map((species) => (
-        <SelectedSpecies
-          key={species.genome_id}
-          species={species}
-          onToggleUse={props.toggleSpeciesUse}
-          onRemove={props.onSpeciesDelete}
-        />
-      ))}
-      {shouldLinkToGenomeBrowser && (
-        <div className={styles.genomeBrowserLinkContainer}>
-          <Link to={urlFor.browser()}>View in Genome Browser</Link>
-        </div>
-      )}
-    </>
-  );
+
+  const selectedSpecies = props.selectedSpecies.map((species) => (
+    <SelectedSpecies
+      key={species.genome_id}
+      species={species}
+      onToggleUse={props.toggleSpeciesUse}
+      onRemove={props.onSpeciesDelete}
+    />
+  ));
+
+  const link = shouldLinkToGenomeBrowser ? (
+    <Link to={urlFor.browser()}>View in Genome Browser</Link>
+  ) : null;
+
+  return <SpeciesTabsWrapper speciesTabs={selectedSpecies} link={link} />;
```

Comment:
`<SpeciesTabsWrapper speciesTabs={selectedSpecies}...`
`<SpeciesTabsWrapper speciesTabs={speciesItems}...`
`<SpeciesTabsWrapper speciesTabs={speciesTabs}...`

It's quite unclear of what speciesTabs can accept. The definition says it's ReactElement<any> with a  FIXME comment next to it. Is it going to be described in this PR?

Reply:
`speciesTabs` are an array of React elements produced by one of the three SelectedSpecies components (`SimpleSelectedSpecies` or `FocusableSelectedSpecies` from the shared components, or the `SelectedSpecies` component from the SpeciesSelector app). I updated the types for `SpeciesTabsWrapper` to reflect that (removed the Fixme, and substituted the `any` type passed to `ReactElement` with more specific types). Don’t know if that helps.